src/utils/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable camelcase */
```

Comment:
is there a reason we are disabling eslint here?

Reply:
Not really, I used a snake case once I had the error but forgot to change it when I later decided to use camelCase. Will remove it shortly. Thanks